SlicerDiffusionQC/GradQC.py

```diff
@@ -390,6 +388,8 @@ def observeStatus(caller,_):
 
   def GUI(self):
 
+      from gradqclib.slicerUserInteraction import slicerGUI
```

Comment:
Should this be `SlicerDiffusionQC.gradqclib.slicerUserInteraction`? Compared with the above, `gradqclib` directory is not at the same level as that of `diffusionqclib`.